python-threatexchange/threatexchange/signal_type/pdq_index.py

```diff
@@ -22,32 +26,37 @@ class PDQIndex(SignalTypeIndex):
     PDQ_CONFIDENT_MATCH_THRESHOLD = 31
     T = IndexT
 
-    def __init__(self, entries: t.Iterable[t.Tuple[str, T]]) -> None:
+    def __init__(
```

Comment:
@schatten FYI I've decided an override flag is the wrong way to do this. I am going to change this in the next commit. Thanks!

Reply:
If you still want two options a simple enum is what is suggested by (internal facebook link): https://www.internalfb.com/intern/wiki/Facebook-code-smells/boolean-arguments/